ocs_ci/ocs/ui/mcg_ui.py

```diff
@@ -370,3 +384,57 @@ def check_obc_option(self, text="Object Bucket Claims"):
         namespace_obj = OCP(kind=constants.NAMESPACE, namespace=sc_name)
         namespaces.append(namespace_obj)
         delete_projects(namespaces)
+
+
+class ObUI(BucketsUI):
+    def __init__(self, driver):
+        super().__init__(driver)
+
+    def delete_object_bucket_ui(self, delete_via, expect_fail):
+        """
+        Delete an Object Bucket via the UI
+
+        delete_via (str): delete via 'OB/Actions' or via 'three dots'
+        expect_fail (str): verify if OB removal fails with proper PopUp message
+        """
+        self.navigate_object_buckets_page()
+        self.delete_resource(delete_via, "ObjectBucket")
+
+        if expect_fail:
+
+            def _check_three_dots_disabled(text):
+                logger.info(text)
+                # when three_dots element is active attribute 'disabled' does not exist
+                assert (
+                    self.get_element_attribute(
+                        self.generic_locators["three_dots"], "disabled"
+                    )
+                    == "true"
+                ), "Three dots are active after OBC removal and OB removal action. Expected - disabled"
+
+                # PopUp is not reachable via Selenium driver. It does not appear in DOM
+                import requests
+                import json
+
+                URL = f"{get_ocp_url()}/locales/resource.json?lng=en&ns=plugin__odf-console"
+
+                cookies = self.driver.get_cookies()
+                session = requests.Session()
+                for cookie in cookies:
+                    session.cookies.set(cookie["name"], cookie["value"])
+
+                popup_str = "The corresponding ObjectBucketClaim must be deleted first."
+                logger.info(f"Send req to {URL}. Get PopUp with {popup_str}")
+
+                resp = session.get(url=URL, verify=False)
+                json_resp = resp.json()
+
+                assert (
+                    popup_str == json_resp[popup_str]
+                ), f"No expected Popup. See full response: \n {json.dumps(json_resp)}"
+
+            sleep(5)
```

Comment:
Pls use explicit wait and avoid hard-coded sleep

Reply:
done